packages/sado-connect/vite.config.ts

```diff
@@ -4,9 +4,27 @@ import { resolve } from "path";
 import dts from "vite-plugin-dts";
 import * as packageJson from "./package.json";
 import commonjs from "@rollup/plugin-commonjs";
+import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill'
+import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfill'
 
 // https://vitejs.dev/config/
 export default defineConfig({
+  optimizeDeps: {
+    esbuildOptions: {
+        // Node.js global to browser globalThis
+        define: {
+            global: 'globalThis'
+        },
+        // Enable esbuild polyfill plugins
+        plugins: [
+            NodeGlobalsPolyfillPlugin({
+                process: true,
+                buffer: true
```

Comment:
Here's the interim solution to the node.js native buffer dependency. @kranthicodes